app/controllers/admin/articles_controller.rb

```diff
@@ -9,29 +10,21 @@ def create
 
     if @article.save(article_params)
       flash[:notice] = "You successfully created a post."
-      redirect_to @article
+      redirect_to admin_articles_path
     else
       flash[:notice] = "Error, please try again."
       render :new
     end
   end
 
-  def show
-    @article = Article.find(params[:id])
-    
-    @article.comments.build
-  end
-
   def index
     @articles = Article.all 
   end
 
   def edit
-    @article = Article.find(params[:id])
   end
 
   def update
-    @article = Article.find(params[:id])
     
     if @article.update(article_params)
       flash[:notice] = "You successfully updated a post."
```

Comment:
Follow the same pattern in all your controllers, the one below in the destroy action would be a good one to use everywhere